warehouse/accounts/interfaces.py

```diff
@@ -179,6 +179,13 @@ def get_webauthn_by_credential_id(user_id, credential_id):
         or None of the user doesn't have a credential with this ID.
         """
 
+    def record_event(user_id, **kwargs):
```

Comment:
Should this interface be more strict than `**kwargs`? I realize it's more or less passing through whatever comes its way to the UserEvent object itself, though a better docstring and explicit signature would be beneficial at this definition.

Reply:
Yep. I made that interface while still hammering the model out; I'll make the keywords explicit and add the appropriate documentation.